python_modules/dagster/dagster_tests/definitions_tests/metadata_tests/test_table_metadata_set.py

```diff
@@ -11,7 +11,7 @@ def error_on_warning():
     raise_exception_on_warnings()
 
 
-def test_table_metadata_set():
+def test_table_metadata_set() -> None:
```

Comment:
Without a type annotation on the function signature, `pyright` will skip analysis of the underlying function. This is because we have `analyzeUnannotatedFunctions` set to `False`: https://github.com/dagster-io/dagster/blob/db84034b1f1173389c239100097fdaa580c47fbd/pyproject.toml#L44-L47

So everything will appear as `Any` in your local editor unless you type the function.

Reply:
Wow, did not know this